backend/de.metas.business/src/main/java/de/metas/bpartner/quick_input/service/BPartnerQuickInputService.java

```diff
@@ -424,36 +428,97 @@ private static BPartnerQuickInputId extractBpartnerQuickInputId(final @NonNull I
 
 	private BPartnerComposite toBPartnerComposite(@NonNull final I_C_BPartner_QuickInput template)
 	{
-		final BPGroupId groupId = BPGroupId.ofRepoIdOrNull(template.getC_BP_Group_ID());
-		if (groupId == null)
+		if (template.getC_BP_Group_ID() <= 0)
+		{
+			throw new FillMandatoryException(I_C_BPartner_QuickInput.COLUMNNAME_C_BP_Group_ID);
+		}
```

Comment:
final BPGroupId groupId = BPGroupId.ofRepoIdOrNull(template.getC_BP_Group_ID());
		if (groupId == null)
		{
			throw new FillMandatoryException("C_BP_Group_ID");
		}

Reply:
Note: I did modify this, but github doesn't show it because my modification is not considered a change, since it was like this on master.